flipperdevices/flipperzero-good-faps
# Picopass: allow write immediately after read

# What's new

- Inspired by trying to clone iClass for people at DefCon and being forced to save the file first.
- Adds Write and Emulate menu options to menu after reading card

# Verification 

- Read card
- See option to write (and it works)
- See option to emulate (and it works)

# Checklist (For Reviewer)
